src/go/k8s/.pre-commit-config.yaml

```diff
@@ -4,7 +4,6 @@ repos:
   - repo: https://github.com/redpanda-data/pre-commit-vectorizedio.git
     rev: 91f6656b0fd532cdf325536a1daf8f28a8fa670c
     hooks:
-      - id: crlfmt
```

Comment:
should this be replaced with gofumpt?

Reply:
There is no pre-commit hook with gofumpt, we will have to create another PR in that repo before changing that here. That's why for now I'm deleting this.

I'll create that issue for the backlog.